changelog

```diff
@@ -1,2 +1,9 @@
 ## v1.0.0
  - Initial tag
+
+## v1.0.1
+ - Add different callbacks for Client resolve/connect/handshake errors
+ - Add support for IPv6 connection
+ - Increase size of websocket write buffer
+ - Add heartbeat mechanism based on websocket ping/pong datagrams
```

Comment:
Are heartbeat timeouts handled here, or should it be handled by users of the this library?

Reply:
Library users specify the duration of the heartbeat period, and the mechanism for detecting disconnections due to heartbeat timeout expiration is implemented external to the library.